modules/preview-react/segmented-control/stories/SegmentedControl.stories.mdx

```diff
@@ -38,62 +37,86 @@ yarn add @workday/canvas-kit-preview-react
 
 ### Basic Example
 
-In a basic example of a `SegmentedControl` there are four icon buttons in group. Each button is
-paired with a tooltip to explain the button function in text. Default values of `SegmentedControl`
-are specified for size (as `medium`) and orientation (as `horizontal`). All buttons should have the
-same type and only one of them can be active at a time.
-
 `SegmentedControl` includes a container `SegmentedControl` component and the following
-subcomponents: `SegmentedControl.List` which contains a `SegmentedControl.Item`.
+subcomponents: `SegmentedControl.List` and `SegmentedControl.Item`.
+
+The example below contains a `SegmentedControl` with four icon-only buttons. Each button is rendered
+using a `SegmentedControl.Item` and is paired with a tooltip describing the button's function. Only
+one button can be active at a time.
 
 <ExampleCodeBlock code={Basic} />
 
-### Variations Example
+Note that you must provide `SegmentedControl.List` with an `aria-label` prop for accessibility
+reasons.
 
-`SegmentedControl` can be used with diferent buttons: text only, text with icon or icon only.
+### Variations
 
-#### Text And Icon
+`SegmentedControl` supports three variations based on whether or not its `SegmentedControl.Item`
+components have an `icon` prop and/or text content: icon-only, text-only, and text-and-icon.
```

Comment:
I updated the `Variations` section to provide more explanation about how to specify each variation.

Reply:
Looks much better! Thank you.